### DISCUSSION TITLE
Regular expression in input.json

### DISCUSSION THREAD

**Discussion by PKUfjh** [deepmodeling/deepmd-kit#1710] [Q&A]
Can I use regular expression in the input.json file? For example,
    "training" : {
    "training_data": {
        "systems":		["../training/C[0-9]*/"],
        "set_prefix":	"set",    
	    "batch_size":	"auto",
	    "_comment":		"that's all"
	},
But it seems to be not working.

**Comment by njzjz** [comment]
> Can I use regular expression in the input.json file?

No.
